### ISSUE TITLE
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting..

### ISSUE THREAD

**Issue by marcinjahn** [unxsist/jet-pilot#40]
I'm trying to run the AppImage release (1.3.0) on Fedora 41.
When running the app graphically, a windows pops up for a brief moment, and disappears, and that's it.
When I run the app from shell, I see those logs:

```
** (jet-pilot:203794): WARNING **: 18:05:01.951: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...

** (jet-pilot:203794): WARNING **: 18:05:26.972: atk-bridge: get_device_events_reply: unknown signature
```

**Comment by DmitryKaznacheev** [comment]
AppImage release (1.31.0), openSUSE Tumbleweed 20241107

(jet-pilot:7018): dbind-WARNING **: 09:23:04.613: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable

** (jet-pilot:7018): WARNING **: 09:23:04.831: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...

**Comment by waldner** [comment]
Same on archlinux (1.31.0)...

```
$ ./jet-pilot_1.31.0_amd64.AppImage 

** (jet-pilot:3289783): WARNING **: 23:39:24.806: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...
```

**Comment by unxsist** [comment]
🚀 This issue is resolved in version 1.31.1 of JET Pilot!
